docs/g_slide_005.md

```diff
@@ -1,7 +1,7 @@
 # Life's reflex glider implemented in Lenia
 
 {:style="text-align:center;"}
-![Life glider, Lenia implementation](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/glaberish/life_glider_lenia.gif)
+![Life glider, Lenia implementation](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/glaberish/life_glider_lenia.gif)
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/g_slide_004) -- [Next slide](https://rivesunder.github.io/yuca/g_slide_006)
+[Previous slide](https://rivesunder.github.io/yuca_docs/g_slide_004) -- [Next slide](https://rivesunder.github.io/yuca_docs/g_slide_006)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.